Libplanet/Action/ActionEvaluation.cs

```diff
@@ -32,7 +36,7 @@ IAccountStateDelta outputStates
         /// <summary>
         /// An action to evaluate.
         /// </summary>
-        public T Action { get; }
+        public IAction Action { get; }
```

Comment:
This change also should be mentioned in the changelog, IMHO?

Reply:
I added changelog and amended the [commit](https://github.com/planetarium/libplanet/pull/367/commits/50a8e4b58b39c3610077d3d1c2dd1f8c34d7eeeb).